spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -910,9 +914,16 @@ public void run() {
 					break;
 				}
 				catch (AuthorizationException ae) {
-					this.fatalError = true;
 					ListenerConsumer.this.logger.error(ae, "Authorization Exception");
-					break;
+					if (this.authExceptionRetryInterval == null) {
+						ListenerConsumer.this.logger.debug("No authExceptionRetryInterval provided");
```

Comment:
You lost the `ae` in the log so we won't know what kind it is.

Reply:
I'll move it from line 914 to both log entries.